libs/integration-tests-ocm/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "@openshift-assisted/integration-tests-ocm",
```

```suggestion
  "name": "@openshift-assisted/integration-tests-ocm",
  "private": true,
```
This package is not intended to be published to npm